iModelCore/ECDb/ECDb/ECSchemas/ECDbMeta.04.00.02.ecschema.xml

```diff
@@ -0,0 +1,669 @@
+        <ECEnumerator name="Integer" value="1281" displayLabel="Integer" />
+        <ECEnumerator name="Long" value="1537" displayLabel="Long" />
+        <ECEnumerator name="Point2d" value="1793" displayLabel="Point2d" />
+        <ECEnumerator name="Point3d" value="2049" displayLabel="Point3d" />
+        <ECEnumerator name="String" value="2305" displayLabel="String" />
+        <ECEnumerator name="IGeometry" value="2561" displayLabel="IGeometry" />
+    </ECEnumeration>
+
+    <!-- Flat list of Custom Attributes without relationships to their container -->
+    <ECEntityClass typeName="CustomAttribute" description="" displayLabel="" modifier="Sealed">
+        <ECCustomAttributes>
+            <ClassMap xmlns="ECDbMap.02.00.00">
+                <MapStrategy>ExistingTable</MapStrategy>
+                <TableName>ec_CustomAttribute</TableName>
+            </ClassMap>
+        </ECCustomAttributes>
+        <ECProperty propertyName="ContainerId"  typeName="int"/>
+        <ECProperty propertyName="ContainerType"  typeName="int"/>
+        <ECProperty propertyName="Ordinal"  typeName="int"/>
+        <ECProperty propertyName="Instance"  typeName="string"/>
+        <ECNavigationProperty propertyName="Class" relationshipName="CustomAttributeClassHasInstance" direction="backward"/>
+    </ECEntityClass>
+    <ECRelationshipClass typeName="CustomAttributeClassHasInstance" modifier="Sealed" strength="referencing">
+        <Source multiplicity="(1..1)" roleLabel="has instance" polymorphic="false">
+            <Class class="ECClassDef"/>
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="is instance of" polymorphic="false">
+            <Class class="CustomAttribute"/>
+        </Target>
+    </ECRelationshipClass>
+
+    <!-- Concrete Custom Attribute instances for property/class/schema -->
+    <ECEntityClass typeName="PropertyCustomAttribute" description="" displayLabel="" modifier="Abstract">
+        <ECCustomAttributes>
+            <View xmlns="ECDbMap.02.00.03">
+                <Query>
+                    SELECT
+                        [ca].[ECInstanceId],
+                        NAVIGATION_VALUE(meta.PropertyCustomAttribute.Property, [ca].[ContainerId]),
+                        NAVIGATION_VALUE(meta.PropertyCustomAttribute.CustomAttributeClass, [ca].[Class].[Id]),
+                        XmlCAToJson([ca].[Class].[Id], [ca].[Instance]) [ValueJson]
+                    FROM [meta].[CustomAttribute] [ca]
+                    WHERE [ca].[ContainerType] = 992
+                    ORDER BY [ca].[Ordinal]
+                </Query>
+            </View>
+        </ECCustomAttributes>
+        <ECNavigationProperty propertyName="Property" relationshipName="PropertyHasCustomAttribute" direction="backward"/>
+        <ECNavigationProperty propertyName="CustomAttributeClass" relationshipName="CustomAttributeClassHasInstanceOnProperty" direction="backward"/>
+        <ECProperty propertyName="ValueJson"  typeName="string"/>

```

Comment:
Should have extended type name of json, maybe then the property name can loose the Json postfix.

Reply:
done